src/Evolution/Systems/Ccz4/Tags.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/Ccz4/TagsDeclarations.hpp"
+#include "Evolution/Tags.hpp"
+#include "Options/Options.hpp"
+
+class DataVector;
+
+namespace Ccz4 {
+namespace Tags {
+/*!
+ * \brief Auxiliary variable which is analytically the spatial derivative of the
+ * natural log of the lapse
+ *
+ * \details If \f$ \alpha \f$ is the lapse, then we define
+ * \f$ A_i = \partial_i ln(\alpha) = \frac{\partial_i \alpha}{\alpha} \f$.
+ */
+template <size_t Dim, typename Frame>
+struct A : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame>;
+};
+
+/*!
+ * \brief Auxiliary variable which is analytically half the spatial derivative
+ * of the conformal spatial metric
+ *
+ * \details If \f$ \widetilde{\gamma}_{ij} \f$ is the conformal spatial metric,
```

Comment:
Would it be ok to use `\bar` instead of `\widetilde` to denote conformal quantities? That's what the XCTS system does, for consistency with the Baumgarte & Shapiro book.